utils/unittest/epoch_builder.go

```diff
@@ -280,7 +287,7 @@ func (builder *EpochBuilder) BuildEpoch() *EpochBuilder {
 	receiptF := ReceiptForBlockFixture(F)
 
 	// build block G
-	// G contains a seal for block E and a receipt for block F
+	// seal for block N , receipt for block n-1
```

Comment:
seems like the removed comment is the one we want here

Reply:
https://github.com/onflow/flow-go/pull/1940/commits/6e64634d3c34f3e167f62d74e1c1cf5e49f5d740